src/materialized/tests/cli.rs

```diff
@@ -24,15 +24,6 @@ fn cmd() -> Command {
 
 #[test]
 fn test_threads() {
-    let assert_fail = |cmd: &mut Command| {
-        cmd.assert().failure().stderr(predicate::str::starts_with(
-            "materialized: '--workers' must be specified and greater than 0",
-        ))
-    };
-    assert_fail(&mut cmd());
-    assert_fail(cmd().arg("-w0"));
-    assert_fail(cmd().env("MZ_WORKERS", "0"));
-
```

Comment:
these two seem like they should still be errors, as materialize won't work with zero workers.

Reply:
Oof thanks for the catch, I was overzealous